src/fluree/server/handlers/transact.clj

```diff
@@ -20,22 +20,16 @@
   [json-txn]
   (crypto/sha2-256 json-txn :hex :string))
 
-(defn normalize-txn
+(defn derive-tx-id
   [txn]
-  (let [txn*  (json-ld/normalize-data
-                txn
-                {:algorithm :basic
-                 :format    :application/json})
-        tx-id (hash-txn txn*)]
-    [tx-id txn*]))
+  (let [canonized  (jld-processor/canonize txn)]
+    (hash-txn canonized)))
 
 (defn queue-consensus
-  [consensus conn watcher ledger tx-id txn* default-context opts]
+  [consensus conn watcher ledger tx-id expanded-txn]
   (let [conn-type       (conn-proto/-method conn)
         ;; initial response is not completion, but acknowledgement of persistence
-        persist-resp-ch (consensus/queue-new-transaction
-                         consensus conn-type ledger tx-id txn* default-context
-                         opts)]
+        persist-resp-ch (consensus/queue-new-transaction consensus conn-type ledger tx-id expanded-txn nil nil)]
```

Comment:
Can you split this up into two lines? (My tall, narrow editor windows thank you in advance!)